osf_pigeon/pigeon.py

```diff
@@ -429,13 +434,13 @@ async def archive(guid):
         if file_count:
             tasks.append(
                 stream_files_to_dir(
-                    from_url=f"{settings.OSF_FILES_URL}v1/resources/{guid}/providers/osfstorage/?zip=",
-                    to_dir=temp_dir,
-                    name="archived_files.zip",
+                    f"{settings.OSF_FILES_URL}v1/resources/{guid}/providers/osfstorage/?zip=",
```

Comment:
why change from kwargs to positional args?

Reply:
just to keep it under 100 horizontal characters.